lib/src/widget/input_history_text_field.dart

```diff
@@ -79,6 +79,12 @@ class InputHistoryTextField extends StatefulWidget {
   // enabled/disabled of input history
   final bool enableHistory;
 
+  /// show history list
+  final bool showHistoryList;
+
+  // enabled/disabled of filter history
+  final bool enableFilterHistory;
+
```

Comment:
I see the same correction in another PR 🤔 
https://github.com/trashfeed/input_history_text_field/pull/9https://github.com/trashfeed/input_history_text_field/pull/9